botcity/maestro/sdk.py

```diff
@@ -120,19 +120,22 @@ def message(self, email: List[str], users: List[str], subject: str, body: str,
     @ensure_implementation()
     @ensure_access_token()
     def create_task(self, activity_label: str, parameters: Dict[str, object],
-                    test: bool = False) -> model.AutomationTask:
+                    test: bool = False, priority: int = 0, min_execution_date=None) -> model.AutomationTask:
         """
         Creates a task to be executed on the BotMaestro portal.
 
         Args:
             activity_label: The activity unique identified.
             parameters: Dictionary with parameters and values for this task.
             test: Whether or not the task is a test.
+            priority: An integer from 0 to 10 to refer to execution priority.
+            min_execution_date: Minimum execution date for the task (dd/mm/yyyy hh:mm).
```

Comment:
This docstring is incorrect. We need a datetime object not a string.
We also need the type annotation.